service/main.toit

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) 2025 Toit contributors.
+// Use of this source code is governed by an MIT-style license that can be found
+// in the LICENSE file.
+
+import encoding.tison
+import system.assets
+import hc-sr04.provider
+
+install-from-args_ args/List:
+  if args.size != 2:
+    throw "Usage: main <trigger> <echo>"
+  trigger := int.parse args[0]
+  echo := int.parse args[1]
+  provider.install --trigger=trigger --echo=echo
+
+install-from-assets_ configuration/Map:
+  trigger := configuration.get "trigger"
+  if not trigger: throw "No 'trigger' found in assets."
+  if trigger is not int: throw "Trigger must be an integer."
```

Comment:
I think I'd prefer to reword the thrown strings for non-ints so we don't have to capitalize the parameter names.